apps/pythonapi/examples/dataset_example.py

```diff
@@ -0,0 +1,54 @@
+# %% [md]
+#
+# # Opening Datasets
+#
+# Vapor supports a variety of scientific data formats.
+# This notebook shows how to open a dataset and query its metadata.
+#
+# %%
+import example_utils
+from vapor import session, renderer, dataset, camera
+
+# %%
+print("Supported dataset types:", dataset.Dataset.GetDatasetTypes())
+
+# %%
+ses = session.Session()
+data = example_utils.OpenExampleDataset(ses)
+
+# Examples of opening real data
+#
+# data = ses.OpenDataset(dataset.WRF, ["data/wrf_out.0001", "data/wrf_out.0002"])
```

Comment:
See comments above. These sample data sets would be more useful for learning if they can be easily accessed by users so that they can interrogate them with their own tools.

Reply:
These are not meant to be real datasets, it is simply meant to show the syntax. For loading datasets all you need is a single line of code demonstrated in these comments. For any further demonstration, the generated demo dataset provides the same functionality as a real one?